ports/zipper/vcpkg.json

```
@@ -0,0 +1,8 @@
+{
+	"name": "zipper",
+	"version-string": "1.0.2",
+	"description": "C++ wrapper around minizip compression library",
+	"dependencies": [

```

```suggestion
       "homepage": "https://github.com/sebastiandev/zipper",
	"dependencies": [
	{
	  "name": "vcpkg-cmake",
	  "host": true
	},
```
The dependent ports should be added here if using the new functions.

For vcpkg-cmake-config, please help confirm if we need to add [vcpkg_cmake_config_fixup](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/ports/vcpkg-cmake-config/vcpkg_cmake_config_fixup.md) in portfiel.cmake.